docs/building-apps/building-64bit-apps.rst

```diff
@@ -0,0 +1,7 @@
+.. _label-building-64bit-apps:
+
+Building 64-bit MPI applications
+================================
+
+Open MPI is 64-bit clean. You should be able to use Open MPI on
+64-bit architectures and operating systems with no difficulty.
```

Comment:
Do we truly still need this file? 64bit is the default, 32bit is the problematic one.

Reply:
I'm fine with removing it.  Anyone else want to keep / delete it?

Reply:
OK

Was 32-bit support officially dropped?  (What was the outcome of the discussion on the mailing list a few weeks ago?)

Reply:
There's this issue where supposedly someone wants to maintain it but hasn't had time, and I'm not sure we're going to delay the release for this indeterminate support - https://github.com/open-mpi/ompi/issues/11409

Reply:
OK, I've removed the sentence(s) about supporting 64-bit platforms in my next commit, since it's the norm now.  If the 32-bit maintenance requires a note in the docs, that can be a separate ticket.